### ISSUE TITLE
RuboCop 0.70.0 has new rule names

### ISSUE THREAD

**Issue by olleolleolle** [varvet/pundit#594]
https://travis-ci.org/varvet/pundit/jobs/536151698 The linting job failed with:

```
.rubocop.yml:81: `Style/TrivialAccessors` is concealed by line 96
Error: The `Layout/IndentHash` cop has been renamed to `Layout/IndentFirstHashElement`.
(obsolete configuration found in .rubocop.yml, please update it)
```

**Comment by rubyrider** [comment]
But Rubocop after 0.69 ended the support for Ruby 2.2. So I prefer locking the rubocop version to 0.68 and fix the existing redundant configuration until the core team decided to end the support for ruby 2.2 as the minimum ruby version.

I have created a pull request considering the above context.
 https://github.com/varvet/pundit/pull/598

**Comment by olleolleolle** [comment]
I'd say, lock to an earlier RuboCop, make it pass.

Then, also, we could have a Ruby version support EOL policy discussion. What's the oldest Ruby we want to support?

**Comment by rubyrider** [comment]
@olleolleolle 29 days ago I have sent an pull request which fix the rubocop version and keep old ruby supported by just updated few rubocop rules!  Would you be interested to review that ?
PR: https://github.com/varvet/pundit/pull/598

Looking forward for your feedback!

**Comment by dgmstuart** [comment]
This specific issue has been fixed by #605 which removed both of these rules from the rubocop config entirely, and locked it to a passing version.
